setuptools/tests/test_editable_install.py

```diff
@@ -61,7 +61,6 @@ def editable_opts(request):
             "Intended Audience :: Developers"
         ]
         urls = {Homepage = "https://github.com"}
-        dependencies = ['importlib-metadata; python_version<"3.8"']
```

Comment:
What about `importlib-metadata>=0.12;python_version<"3.8"` in `setuptools/tests/config/test_pyprojecttoml.py` and `importlib-metadata==6.0.0 ; python_version<"3.8"` in `setuptools/tests/test_core_metadata.py` ? I'm not sure whether using a marker is an important part of the test, or if it's just an outdated requirement.

Reply:
Oh. I was unaware of #4096. I'll defer to your work since it landed first.

You have good insight here - these markers are potentially still relevant regardless what version of Python is supported by Setuptools (e.g. one could still distribute a package supporting older Pythons using the latest Setuptools).